components/brave_wallet_ui/utils/format-prices.ts

```diff
@@ -0,0 +1,124 @@
+import BigNumber from 'bignumber.js'
```

Comment:
needs license header

Reply:
bignumber.js should also be pinned to an exact version in package.json

Reply:
@diracdeltas Pinned to exact version